### DISCUSSION TITLE
Documentation rework

### DISCUSSION THREAD

**Discussion by samuel-jimenez** [actionless/pikaur#797] [Ideas]
What are your thoughts on splitting up the documentation? 

I wonder if it might flow better if, instead of one monolithic readme, it is split into more context-appropriate sections?

Maybe something akin to `pacman`'s strategy, where you can `man pacman` to get basic operation and command-line arguments, but  `man pacman.conf` will tell about the config file?

**Comment by actionless** [comment]
yup, would make sense 👍

**Reply by actionless** [reply]
(but don't do it as part of your current PR)

**Comment by samuel-jimenez** [comment]
Here are my thoughts for the process. Clearly GitHub uses `README.md`, so that file needs to be kept.

1. Split the Configuration section of the current readme to be accessed by `man pikaur.conf`
2.  Could `args.py` be leveraged to produce  `man pikaur`?
3. Access to the readme through  `man pikaur.install` or something similar

**Reply by actionless** [reply]
1) yup
2) they could but there is no need for that - as same information is accessible with --help 
3) installation part is not needed at all in the generated doc - i'd rather write some transforming expression to cut it out while converting md to man

**Reply by samuel-jimenez** [reply]
Sounds good. One thing: --help with no operation will not list any options. Is this intentional behavior?

**Reply by actionless** [reply]
did you checked `pacman`?

**Reply by samuel-jimenez** [reply]
Right, but pacman enumerates top-level options in the man page.

**Reply by actionless** [reply]
that's a different matter:
in the manpage pacman options have much longer description for each option than in pacman --help

**Reply by actionless** [reply]
and top-level flags (ie not options) unique to pikaur (-P and -G) are documented in its manpage
